networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRangeMessageHandler.java

```diff
@@ -263,6 +258,9 @@ SafeFuture<Optional<BlobSidecar>> loadNextBlobSidecar() {
             .thenCompose(
                 block -> {
                   maybeCurrentBlock = block;
+                  final MiscHelpersDeneb miscHelpersDeneb =
+                      spec.atSlot(currentSlot.get()).miscHelpers().toVersionDeneb().orElseThrow();
```

Comment:
I think we are not guaranteed here to be inside Deneb. Am I right?
If yes, we need quiet skip here to move to the next block

Reply:
At this stage we're sure the request epoch is after `MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS`. We perform `checkRequestInMinEpochsRange(requestEpoch) && !checkBlobSidecarsAreAvailable(earliestAvailableEpoch, requestEpoch)` in `onIncomingMessage`

Reply:
Oh, my bad, yeah